meinberlin/templates/base.html

```diff
@@ -47,7 +53,11 @@
 <body>
     <a href="#main" class="visually-hidden-focusable">{% translate "Skip to content "%}</a>
     {% block header %}
-      {% include 'header.html' %}
+    {% if '/dashboard' in request.path %}
+        {% userindicator %}
+    {% else %}
+        {% include 'header.html' %}
+    {% endif %}
```

Comment:
it's not confirmed if we need their header and footer in the dashboard yet, if so that makes the solution above non viable but also I don't know why we would need a platform header in a CMS

Reply:
similar here, make use of the blocks for overriding between templates